Fix duplicate Gemfile entry warning

zzak opened the pull request:
```
Your Gemfile lists the gem rake (>= 13) more than once.
You should probably keep only one of them.
Remove any duplicate entries and specify the gem only once.
While it's not a problem now, it could cause errors if you change the version of one of them later.
```

yahonda commented on .github/workflows/mdl.yml:
```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
-      BUNDLE_ONLY: mdl
+      BUNDLE_ONLY: default mdl
```
Let me confirm what `BUNDLE_ONLY: default` means. If it installs every gem file in the Gemfile I do not think we need to have this line. Because it installs every gems anyway.

zzak replied:
@yahonda Thanks for your review! It seems we can use `BUNDLE_INCLUDE` instead, see [f8d35ee](https://github.com/rails/rails/pull/47855/commits/f8d35ee5d31a728c4104c76a05a7fa561e748e28).

zzak replied:
Ahh, actually include is redundant so we can just remove the environment variable from the action.

zzak committed:
Fix duplicate Gemfile entry warning

Your Gemfile lists the gem rake (>= 13) more than once.
You should probably keep only one of them.
Remove any duplicate entries and specify the gem only once.
While it's not a problem now, it could cause errors if you change the version of one of them later.
